docs/notebooks/tutorials/adv_newcv.ipynb

```diff
@@ -5,13 +5,695 @@
+    "\n",
+    "In our example the characteristic parameters for the `AutoEncoderCV` are just the `encoder_layer` (compulsory) and the `decoder_layer` (optional). \n",
+    "\n",
+    "To stay as user-friendly as possible, in `mlcolvar`, we always try to give meaningful and intelligible names to the parameters. Besides that, it is also a good practice to provide a complete docstring for the `__init__` method, explening more in detail what each parameter is actually doing in the model.\n",
+    "\n",
+    "Note that once the undelying architecture for the model is defined the `in/out_features` can be directly inferred from there.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class AutoEncoderCV(BaseCV, lightning.LightningModule):\n",
+    "    BLOCKS = ['norm_in','encoder','decoder']  \n",
+    "\n",
+    "    def __init__(self,\n",
+    "# ================================================ LOOK HERE 0.0 ================================================   \n",
+    "                encoder_layers : list, \n",
+    "                decoder_layers : list = None, \n",
+    "                options : dict = None, \n",
+    "                **kwargs):\n",
+    "        \"\"\"\n",
+    "        Train a CV defined as the output layer of the encoder of an autoencoder model (latent space). \n",
+    "        The decoder part is used only during the training for the reconstruction loss.\n",
+    "\n",
+    "        Parameters\n",
+    "        ----------\n",
+    "        encoder_layers : list\n",
+    "            Number of neurons per layer of the encoder\n",
+    "        decoder_layers : list, optional\n",
+    "            Number of neurons per layer of the decoder, by default None\n",
+    "            If not set it takes automaically the reversed architecture of the encoder\n",
+    "        options : dict[str,Any], optional\n",
+    "            Options for the building blocks of the model, by default None.\n",
+    "            Available blocks: ['norm_in', 'encoder','decoder'].\n",
+    "            Set 'block_name' = None or False to turn off that block\n",
+    "        \"\"\"\n",
+    "        super().__init__(in_features=encoder_layers[0], out_features=encoder_layers[-1], **kwargs)\n",
+    "        \n",
+    "# ================================================ LOOK HERE 0.0 ================================================   \n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Parse options and parameters\n",
+    "The different options in the `options` dictionary are parsed using the `BaseCV.parse_options` function.\n",
+    "\n",
+    "Here we also initialize what is needed from the input parameters. In our case for example we specify that, whenever `decoder_layer` is not specified, it should be the reversed `encoder_layer`."
```

Comment:
I would keep it because:

"This command (`parse_options`) is required as it also initializes defaults whenever specific options entries are not specified and checks that the given options make sense with the CV at hand."

I think it's better to specify it. Maybe you can come up with a better wording though